config/esp32/components/chip/Kconfig

```diff
@@ -1176,8 +1176,8 @@ menu "CHIP Device Layer"
     menu "Commissioning Window Options"
         config CHIP_DISCOVERY_TIMEOUT_SECS
             int "Commissioning Window Timeout in seconds"
-            range 180 900 
-            default 900 
+            range 180 172800
```

Comment:
This range should be possible only when the extended announcement is enabled

Reply:
made the change